azureenergylabelercli/azureenergylabelercli.py

```diff
@@ -113,30 +112,36 @@ def get_arguments():
     parser.add_argument('--frameworks',
                         '-f',
-                        default=os.environ.get('AZURE_LABELER_FRAMEWORKS', DEFAULT_DEFENDER_FOR_CLOUD_FRAMEWORKS),
+                        default=os.environ.get('AZURE_LABELER_FRAMEWORKS', ['Azure Security Benchmark']),
                         type=comma_delimited_list,
-                        help='The list of applicable frameworks: \
+                        help='The comma delimited list of applicable frameworks: \
                                     ["Azure Security Benchmark", "Azure CIS 1.1.0"], '
-                             'default=["Azure Security Benchmark"]')
+                             'default=["Azure Security Benchmark"]\n'
+                             'example="Azure Security Benchmark,Azure CIS 1.1.0"')
     subscription_list = parser.add_mutually_exclusive_group()
     subscription_list._group_actions.append(single_subscription_action)  # pylint: disable=protected-access
     subscription_list.add_argument('--allowed-subscription-ids',
                                    '-a',
                                    required=False,
                                    default=os.environ.get('AZURE_LABELER_ALLOWED_SUBSCRIPTION_IDS'),
                                    type=comma_delimited_list,
-                                   help='A list of Azure Subscription IDs for which an energy label will be produced. '
-                                        'Mutually exclusive with '
-                                        '--denied-subscription-ids and --single-subscription-id arguments.')
+                                   help=('A comma delimited list of Azure Subscription IDs'
+                                         ' for which an energy label will be produced. '
+                                         'Mutually exclusive with '
+                                         '--denied-subscription-ids and --single-subscription-id arguments.\n'
+                                         'example='
+                                         '"00000000-0000-0000-0000-000000000000,00000000-0000-0000-0000-000000000001"'))
     subscription_list.add_argument('--denied-subscription-ids',
                                    '-d',
                                    required=False,
                                    default=os.environ.get('AZURE_LABELER_DENIED_SUBSCRIPTION_IDS'),
                                    type=comma_delimited_list,
-                                   help='A list of Azure Subscription IDs that will '
-                                        'be excluded from producing the energy label. '
-                                        'Mutually exclusive with '
-                                        '--allowed-subscription-ids and --single-subscription-id arguments.')
+                                   help=('A list of Azure Subscription IDs that will '
```

Comment:
Maybe mention the comma delimited situation like in the above cases?

Reply:
Nice catch